README.md

```diff
@@ -328,141 +331,151 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
+                    Setpoint item=LandroidScheduleTuesdayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleTuesdayEdgecut
+                }
+                Frame label="Schedule Wednesday" {
+                    Setpoint item=LandroidScheduleWednesdayStartHour minValue=0 maxValue=23 step=1
+                    Setpoint item=LandroidScheduleWednesdayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleWednesdayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleWednesdayEdgecut
+                }
+                Frame label="Schedule Thursday" {
+                    Setpoint item=LandroidScheduleThursdayStartHour minValue=0 maxValue=23 step=1
+                    Setpoint item=LandroidScheduleThursdayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleThursdayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleThursdayEdgecut
+                }
+                Frame label="Schedule Friday" {
+                    Setpoint item=LandroidScheduleFridayStartHour minValue=0 maxValue=23 step=1
+                    Setpoint item=LandroidScheduleFridayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleFridayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleFridayEdgecut
+                }
+                Frame label="Schedule Saturday" {
+                    Setpoint item=LandroidScheduleSaturdayStartHour minValue=0 maxValue=23 step=1
+                    Setpoint item=LandroidScheduleSaturdayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleSaturdayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleSaturdayEdgecut
+                }
+                Frame label="Schedule Sunday" {
+                    Setpoint item=LandroidScheduleSundayStartHour minValue=0 maxValue=23 step=1
+                    Setpoint item=LandroidScheduleSundayStartMinutes minValue=0 maxValue=45 step=15
+                    Setpoint item=LandroidScheduleSundayDuration minValue=15 maxValue=1425 step=15
+                    Switch item=LandroidScheduleSundayEdgecut
+                }
             }
+            Slider item=LandroidRainDelay minValue=0 maxValue=750 step=30
+        }
+        Frame label="Statistic" {
+            Text item=LandroidTotalTime
+            Text item=LandroidTotalDistance label="Total Distance [%.2f km]"
+            Text item=LandroidTotalBladeTime
         }
-        Setpoint item=LandroidRainDelay minValue=0 maxValue=750 step=30
-    }
-    Frame label="Statistic" {
-        Text item=LandroidTotalTime
-        Text item=LandroidTotalDistance label="Total Distance [%.2f km]"
-        Text item=LandroidTotalBladeTime
-    }
-    Frame label="Orientation" {
```

Comment:
'Orientation' is missing

Reply:
OK, had this for testing under "move" in my sitemap and not migrated back :-(

This is now fixed.